tests/upgrades/test_satellite_maintain.py

```diff
@@ -33,68 +35,14 @@ class TestSatelliteMaintain:
-        version list.
-
-        :return: satellite_version, upgradeable_version, major_version_change
-        """
-
-        cmd = "rpm -q satellite > /dev/null && rpm -q satellite --queryformat=%{VERSION}"
-        # leaving this section as-is for now but this could be refactored to use sat_obj.version
-        satellite_version = sat_obj.execute(cmd)
-        if satellite_version.status == 0:
-            satellite_version = satellite_version.stdout
-        else:
-            return [], [], None, None
-        satellite_maintain_version = sat_obj.execute(
-            "satellite-maintain upgrade list-versions --disable-self-upgrade"
-        )
-        upgradeable_version = [
-            version for version in satellite_maintain_version.stdout if version != ''
-        ]
-        version_change = 0
-        for version in upgradeable_version:
-            version_change += int(version.split('.')[0])
-        if version_change % 2 == 0:
-            major_version_change = False
-            y_version = ''
-        else:
-            major_version_change = True
-            y_version = list(set(satellite_maintain_version) - set(satellite_version))[0].split(
-                '.'
-            )[-1]
-
-        return satellite_version, upgradeable_version, major_version_change, y_version
-
-    @staticmethod
-    def version_details(
-        satellite_version, major_version_change, y_version, upgrade_stage="pre-upgrade"
-    ):
-        """
-        This function is used to update the details of zstream upgrade and
-        next version upgrade
-        :param str satellite_version: satellite version would be like 6.5.0, 6.6.0, 6.7.0
-        :param bool major_version_change: For major version upgrade like 6.8 to 7.0, 7.0
-        to 8.0 etc, then major_version_change would be True.
-        :param str y_version: y_version change depends on major_version_change
-        :param str upgrade_stage: upgrade stage would be pre or post.
-        :return: zstream_version, next_version
+        Obtains upgradable version list by satellite-maintain.
+        :return: upgradeable_versions
         """
-
-        major_version = satellite_version.split('.')[0:1]
-        if major_version_change:
-            major_version = [int(major_version[0]) + 1].append(y_version)
-        else:
-            y_version = int(satellite_version.split('.')[0:2][-1])
-        zstream_version = ''
-        if upgrade_stage == "pre-upgrade":
-            major_version.append(str(y_version + 1))
-            zstream_version = ".".join(satellite_version.split('.')[0:2]) + ".z"
-        else:
-            major_version.append(str(y_version))
-            major_version.append("z")
-        next_version = ".".join(major_version)
-        return zstream_version, next_version
+        cmd = "satellite-maintain upgrade list-versions --disable-self-upgrade"
+        list_versions = sat_obj.execute(cmd).stdout
+        regex = re.compile(r'^\d+\.\d+')
```

Comment:
At line number 41 & 43, use a single quote or double quote to maintain consistency